app/socket-apis/__tests__/get-points-of-ids.js

```diff
@@ -0,0 +1,123 @@
+// https://github.com/EnCiv/civil-pursuit/issues/203
+import { MongoMemoryServer } from 'mongodb-memory-server'
+import { Mongo } from '@enciv/mongo-collections'
+import { ObjectId } from 'mongodb'
+import Points from '../../models/points'
+import getPointsOfIds from '../get-points-of-ids'
+
+let memoryServer
+let db
+
+const synuser = { synuser: { id: 'user1' } }
+
+beforeAll(async () => {
+  memoryServer = await MongoMemoryServer.create()
+  const uri = memoryServer.getUri()
+  await Mongo.connect(uri)
+  db = Mongo.db
+  Points.setCollectionProps()
+})
+
+afterAll(async () => {
+  await Mongo.disconnect()
+  await memoryServer.stop()
+})
+
+beforeEach(async () => {
+  jest.spyOn(console, 'error').mockImplementation(() => {})
+})
+
+afterEach(async () => {
+  console.error.mockRestore()
+})
+
+// Test case 1: Nothing found (expect empty arrays)
+test('Nothing found', async () => {
+  const callback = jest.fn()
+
+  // Call getPointsOfIds with `this` bound to `synuser`
+  await getPointsOfIds.call(synuser, [new ObjectId(), new ObjectId()], callback)
```

Comment:
should be new ObjectId().toString().  the ids should all be the string from of an object id.

The api, is rightly casing each id into an ObjectId, and maybe it will also cast objects into objects, but these tests should try to emulate the real thing. 

The reason is that when ObjectIds go through socket.io between the browser and the server they get turned into strings.